packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -39,32 +39,30 @@ class RawKeyEventDataLinux extends RawKeyEventData {
   /// (GLFW, GTK, QT, etc) may have a different key code mapping.
   final KeyHelper keyHelper;
 
+  /// An int with one or more unicode scalar values in a single keystroke. Two values are handled
+  /// at most. Otherwise, an assertion will be raised. This is typically the character
+  /// that [keyCode] would produce without any modifier keys. For dead keys, it is typically
+  /// the diacritic it would add to a character. Defaults to 0, asserted to be not null.
+  final int unicodeScalarValuesProduced;
```

Comment:
I know, I'm the one that suggested it, but would it be better to just be `unicodeScalarValues`?  Reading it now, I feel like the "Produced" is redundant: we know it's being produced.

Reply:
Done